### DISCUSSION TITLE
Mixing Analog and Digital 74HC4067

### DISCUSSION THREAD

**Discussion by runinit** [tttapa/Control-Surface#896] [Q&A]
Somewhat related to this library and also more general towards the entire design discussion for midi controllers.

I'm working on a design with the Pico and I ran out of pins. Initially I had two separate chains of 74HC4067s. One for analog and one for digital.. but I'm wondering if you can mix analog and digital with chained ICs provided we dedicated EN from each IC and the entire IC is dedicated to either analog or digital.

If I can do this.. does control surface technically support this?

**Comment by Trickster-git** [comment]
Yes, it possible i use 74HC4067 for buttons and for pots. it must be separate decision for analog and digital. 74HC4067 can make from 1 controller pin to 16 pins and need use 4 pin to control 74HC4067.
